src/components/tabs/index.css

```diff
@@ -0,0 +1,19 @@
+.styled-nav {
+    width: 200px;
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    padding: 8px 12px;
+  }
+  
+  .tab-button {
```

Comment:
tab-button class şu ama component'in tabs ismi olursa her zaman ilk styled-tabs olacak 

örneğini
styled-tabs
sonra
styled-tabs-button
sonra şeyler gibi